.template/spec/base/config/environments/production_spec.rb

```diff
@@ -23,4 +31,12 @@ def mailer_default_url_config
       }
     EOT
   end
+  
+  def i18n_config
+    <<~EOT
+      config.i18n.available_locales = ENV.fetch('AVAILABLE_LOCALES').split(', ')
```

Comment:
@andyduong1920 why we need a _space_ after `, ` here?

It means the `AVAILABLE_LOCALES="'en, th, se"` should be defined with space after each comma?

Reply:
Yeah, correct @byhbt ✅